nixos/modules/services/networking/zerotierone.nix

```diff
@@ -57,6 +57,7 @@ in
 
     # ZeroTier does not issue DHCP leases, but some strangers might...
     networking.dhcpcd.denyInterfaces = [ "zt*" ];
+    networking.notNetworkdManagedInterfaces = "zt*";
```

Comment:
how do adresses appear on these interfaces? Is that all managed by the zerotierone daemon, or would there be a usecase where I'd do that with networkd? In that case, maybe add just a `mkDefault` here…

Reply:
Ok, done and rebased with current master to avoid conflicts.